src-tauri/src/manager.rs

```diff
@@ -111,17 +119,36 @@ fn start_watcher(name: &'static str, tx: Sender<WatcherMessage>) {
         })
         .unwrap();
 
-        // Wait for the child to exit
-        let output = child
-            .unwrap()
-            .wait_with_output()
-            .expect("failed to wait on child");
-
-        // Send the process output to the manager
-        tx.send(WatcherMessage::Stopped {
-            name: name.to_string(),
-            output,
-        })
```

Comment:
Just an idea: maybe we could keep this, since it's pretty clean, and instead return the process PID in `WatcherMessage::Started` to be stored in `ManagerState`, which is then killed from the main thread?

Might remove the need for polling the state.

Reply:
This would greatly simplify everything. Let me get started on it and see!

Reply:
On the off chance, the process exits and it's not reported to `start_manager` sending a SIGTERM could potentially kill a random a process. "The chances are near zero", Oppenheimer reference. I think the reporting is near realtime but there could be a chance? I still don't think it's an issue really just a random thought.